.flake8

```diff
@@ -1,3 +1,3 @@
 [flake8]
-ignore = 
-    F841
+exclude = docs/conf.py
```

Comment:
Flake 8 is not recognizing jinja2 syntax.

Reply:
yep ... because this file is not inside the template folder ... so the jinja2 doesn't apply here.

but instead of `.flake8` add these configurations into setup.cfg please :)